source/install/install-ubuntu-1404-postgresql.rst

```diff
@@ -0,0 +1,79 @@
+.. _install-ubuntu-1404-postgresql:
+
+Installing PostgreSQL Database Server
+=====================================
+
+Install and set up the database for use by the Mattermost server. You can install either PostgreSQL or MySQL. To install MySQL, see :ref:`install-ubuntu-1404-mysql`
+
+Assume that the IP address of this server is 10.10.10.1
+
+**To install PostgreSQL on Ubuntu Server 14.04:**
+
+1. Log into the server that will host the database and issue the following command:
+
+  ``sudo apt-get install postgresql postgresql-contrib``
+  
+  When the installation is complete, the PostgreSQL server is running, and a Linux user account called *postgres* has been created.
+
+2. Log into the *postgres* account. 
+
+  ``sudo --login --user postgres``
+
+3. Start the PostgreSQL interactive terminal.
+
+  ``psql``
+
+4.  Create the Mattermost database.
+
+  ``postgres=# CREATE DATABASE mattermost;``
+
+5.  Create the Mattermost user 'mmuser'.
```

Comment:
I changed it to italics. See commit https://github.com/mattermost/docs/pull/742/commits/368e146fe2bd51cadd385df68d9ae1588940b212